docs/overview/getting-started/avalanche-platform.md

```diff
@@ -4,40 +4,53 @@ description: Avalanche features 3 built-in blockchains which includes Exchange C
 
 # Avalanche Platform
 
-Avalanche features 3 built-in blockchains: [**Exchange Chain (X-Chain)**](#exchange-chain-x-chain), 
-[**Platform Chain (P-Chain)**](#platform-chain-p-chain), and
-[**Contract Chain (C-Chain**)](#contract-chain-c-chain). All 3 blockchains are 
-[validated](../../nodes/validate/staking.md) and 
-secured by all Avalanche validators which is also referred as the Primary Network. The Primary 
-Network is a special [Subnet](../../subnets/README.md), and all members of all custom Subnets must
-also be a member of the Primary Network by staking at least 2,000 AVAX.
+Avalanche is a heterogeneous network of blockchains. As opposed to homogeneous networks, where
+all applications reside in the same chain, heterogeneous networks allow separate chains to be
+created for different applications.
+
+Avalanche is an ecosystem that allows users to execute smart contracts, interact with validators and
```

Comment:
This paragraph is a bit wordy and confusing. I'd suggest rewriting it. Don't start both sentences with Avalanche and try using active voice.

Reply:
Reworded